threading_utils.py

```diff
@@ -0,0 +1,4 @@
+def propagate_exceptions(futures):
+    # Calling result() on a future whose execution raised an exception will propagate the exception to the caller
+    [future.result() for future in futures]
+    
```

Comment:
would be great if we could get rid of the github no newline indication
(if you have no character at line 4, I think it will not have it)

Reply:
fixed ... https://github.com/databrickslabs/migrate/pull/142/commits/0c2314d44698baad41fad13298cef7e8c53834fe